poseidon/src/permutation.rs

```diff
@@ -1,105 +1,107 @@
-//! The permutation module contains the function implementing the permutation used in Poseidon
-
-use crate::constants::SpongeConstants;
-use crate::poseidon::{sbox, ArithmeticSpongeParams};
-use ark_ff::Field;
-
-fn apply_mds_matrix<F: Field, SC: SpongeConstants>(

```

Comment:
Regarding this diff: this is caused by whitespace format most likely. If you click on "hide whitespace" in github panel above, the whole diff disappears.

(see the sreenshot)
![screen_2024-09-24_003](https://github.com/user-attachments/assets/834025e3-2431-47cd-9a9d-9218b86c9da6)

Reply:
If someone can run `cargo fmt` on their machine and check if they get the same diff that'd be great. In case it's reproducible I'm pro leaving it, since our policy seems to be relying on automatic tools regarding styling.

Reply:
I actually get a lot of `o1vm` op-program files with hash digests that I don't think should be there. 
<img width="209" alt="Captura de pantalla 2024-09-24 a las 20 00 24" src="https://github.com/user-attachments/assets/2a9e25cb-335b-4b58-b5fa-1ea25f37eda5">
<img width="208" alt="Captura de pantalla 2024-09-24 a las 20 00 30" src="https://github.com/user-attachments/assets/6614cc4d-70e0-4839-9f5a-081f52cf9182">

Reply:
LGTM.
Process:
- checkout on this branch.
- take `develop` version of the file.
- copy in this branch
- check there are actual changes (there are)
- run `cargo +nightly fmt`.
- check `git diff` is empty
- enjoy and approve PR.